lib/platform/tenant-config.js

```diff
@@ -11,6 +11,7 @@ const defaultConfig = {
     showPerms: false,
     hasAllPerms: false,
     languages: ['en'],
+    useSecureTokens: false,
```

Comment:
Why default to `false`? This will hard-coded to `true` in Sunflower, if not Ramsons.

Reply:
I set `false` because it wasn't present in here before. But since we have it as `true` in platform config then it doesn't hurt to set to `true` here as well